cpp/src/io/utilities/hostdevice_vector.hpp

```diff
@@ -30,6 +30,13 @@
 
 #include <variant>
 
+inline bool hostdevice_vector_uses_pageable_buffer()
```

Comment:
This change isn't relevant, is it? Just a cleanup that was lumped in?

Reply:
that's right